frontend/src/pages/Login.js

```diff
@@ -19,14 +17,16 @@ const SignUpSchema = (invalidEmail, noEmail, invalidPassword) => (Yup.object().s
         .required()
 }))
 
-const Login = () => {
+const Login = ({ setTokenInState }) => {
```

Comment:
Itt is használhatnánk a hook-ot és akkor nem kellene a props-on keresztül átadogatni.